docs/assets/js/main.js

```diff
@@ -7,12 +7,13 @@ import Table from '@cfpb/cfpb-tables/src/Table';
 const anchors = new AnchorJS();
 // Add anchors to all headings (except page title headings)
 anchors.add( 'h2:not(.title), h3, h4, h5' );
-// Ensure there are no anchors in the live code examples
+// Ensure there are no anchors in inconvenient places
 anchors.remove( `
   .live-code-example h2,
   .live-code-example h3,
   .live-code-example h4,
   .live-code-example h5,
+  .o-expandable_label,
```

Comment:
This is anywhere an expandable is used. Having the anchor link added to expandable labels is not ideal IMHO because it interferes with the opening/closing functionality whenever an expandable label is clicked.

Reply:
Oh and I swapped the expandables on this page for details/summary elements but the open/close arrows weren't playing nice with the wide width of our content section. I was weary of tweaking that CSS until the side nav is finalized so I just reverted back to expandables for now.